index.js

```diff
@@ -89,12 +89,15 @@ module.exports = class GameActivityToggle extends Plugin {
 
                 r.props.children.unshift(React.createElement(r.props.children[0].type, {
                     icon: () => showCurrentGame ? enabledIcon('20') : disabledIcon('20'),
+                    'aria-label': `${showCurrentGame ? 'Hide' : 'Show'} Game Activity`,
+                    'aria-checked': `${showCurrentGame ? 'false' : 'true'}`,
+                    role: "switch",
```

Comment:
useless
```suggestion
```

Reply:
Neither of the attributes are added to the button AFAICT, but they do not seem to affect the functionality of this particular button. Tidier without for sure.

Reply:
![image](https://user-images.githubusercontent.com/31005896/92417889-fce34d80-f164-11ea-82de-7e54d3f7fddd.png)